packages/preload/src/modules/workspace.ts

```diff
@@ -54,10 +55,22 @@ export async function hasNodeModules(_path: string) {
   return exists(nodeModulesPath);
 }
 
-export async function getProjectThumbnail(projectPath: string, scene: Scene): Promise<string> {
+export async function getProjectThumbnailPath(_path: string) {
+  const editorHomePath = await getEditorHome(_path);
+  return path.join(editorHomePath, 'images', 'project-thumbnail.png');
+}
+
+export async function getProjectThumbnailAsBase64(
+  projectPath: string,
+  scene: Scene,
+): Promise<string> {
   try {
-    if (!scene.display?.navmapThumbnail) return DEFAULT_THUMBNAIL;
-    const thumbnailPath = path.join(projectPath, scene.display.navmapThumbnail);
+    if (scene.display?.navmapThumbnail) {
+      const thumbnailPath = path.join(projectPath, scene.display.navmapThumbnail);
+      return (await fs.readFile(thumbnailPath)).toString('base64');
```

Comment:
Maybe here we could try/catch in case the file does not exist, fallback to the automatic screenshot instead of the static image.